Specs/Scene/OctahedralProjectedCubeMapSpec.js

```diff
@@ -113,7 +113,7 @@ describe(
         sampleOctahedralMap(octahedralMap, directionFlipY, lod, function (
           octahedralMapColor
         ) {
-          return expect(cubeMapColor).toEqualEpsilon(octahedralMapColor, 5);
+          return expect(cubeMapColor).toEqualEpsilon(octahedralMapColor, 7);
```

Comment:
I remember talking about this offline. Once https://github.com/CesiumGS/cesium/pull/9040#discussion_r458780605 is addressed it should be fine to use `5` again.

Reply:
That probably explains that. Ok will do!

Reply:
Update to this: the bug currently seems to be with Filament generating an invalid KTX file. [More Here](https://github.com/KhronosGroup/KTX-Software/issues/278)